doc/source/deployment/setup-deployer.rst

```diff
@@ -68,22 +68,37 @@ Deployer. Set up your workspace with the following steps:
   export SOCOK8S_WORKSPACE_BASEDIR=~
 
 
-Cloning the repository
-----------------------
+Installing the SUSE Containerized OpenStack software
+----------------------------------------------------
 
-To get started, clone the `socok8s GitHub repository <https://github.com/SUSE-Cloud/socok8s>`_.
-This repository uses submodules, so you need to get all the code to make sure
-the playbooks work.
+There are several ways to install the SUSE Containerized OpenStack software.
 
-::
+1. Install from the openSUSE repository including required dependencies:
 
-   git clone --recursive https://github.com/SUSE-Cloud/socok8s.git
+   a. sudo zypper addrepo \\
+      https://download.opensuse.org/repositories/Cloud:/socok8s/openSUSE_Leap_15.0/ socok8s
```

Comment:
You need to add `-f` or `zypper refresh`

Reply:
I think it's not technically necessary, as following `zypper install` should refresh new repo as well

Reply:
ah, I thought it only did that when the last refresh was a certain age. It makes sense to do it on new repos though.

Reply:
done, refresh added